pandas-stubs/util/_tester.pyi

```diff
@@ -1 +1 @@
-def test(extra_args=...) -> None: ...
+def test(extra_args: list[str] | None = None, run_doctests: bool = False) -> None: ...
```

Comment:
Realigned with main branch of pandas.

Reply:
unsure whether we should include the default arguments here.  We haven't in the past.

Reply:
Mistake from the copy paste, will fix